turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksSession.java

```diff
@@ -665,6 +681,16 @@ public void setScreenshotsEnabled(boolean enabled) {
     }
 
     /**
+     * <p>Determines whether WebViews can be refreshed by pulling/swiping from the top
+     * of the WebView. Default is true.</p>
+     *
+     * @param enabled If true pulling to refresh the WebView is enabled
+     */
+    public void setPullToRefreshEnabled(boolean enabled) {
```

Comment:
I think this is 👍 as is. This strikes me as something that would be set once per session, so it makes sense as a simple configuration. I don't foresee many scenarios where someone would want to config PTR on a per-visit basis.